FE/src/components/interviewPage/InterviewMain/InterviewQuestion.tsx

```diff
@@ -0,0 +1,46 @@
+import Typography from '@foundation/Typography/Typography';
+import { css } from '@emotion/react';
+import { theme } from '@styles/theme';
+type InterviewQuestionType = {
+  question: string;
+};
+
+const InterviewQuestion: React.FC<InterviewQuestionType> = ({ question }) => {
+  return (
+    <div
+      css={css`
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        position: absolute;
```

Comment:
```suggestion
        position: fixed;
```

아까 말씀드린 변경사항입니다! 동영상 영역이 스크롤되어도 질문이 고정되게 변경해주세요!

Reply:
b66305557d91d3c01926f59a69888a4fa5e7900f

반영완료!!